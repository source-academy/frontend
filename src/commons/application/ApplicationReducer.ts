import { Action, Reducer } from 'redux';

import { ApplicationState, defaultApplication } from './ApplicationTypes';

export const ApplicationReducer: Reducer<ApplicationState> = (
  state = defaultApplication,
  action: Action
) => {
  return state;
};
