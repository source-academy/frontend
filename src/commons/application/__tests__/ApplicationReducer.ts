import { ApplicationReducer } from '../ApplicationReducer'; // EDITED

const initialState = ApplicationReducer(undefined!, { type: '*' });

test('initial state should match a snapshot', () => {
  expect(initialState).toMatchSnapshot();
});
