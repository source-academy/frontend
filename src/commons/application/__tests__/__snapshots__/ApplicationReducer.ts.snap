// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`initial state should match a snapshot 1`] = `
Object {
  "environment": "test",
}
`;
